content/ja/docs/contributing/sig-practices.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 承認者およびメンテナーのための SIG のプラクティス
+linkTitle: SIG のプラクティス
+description: 承認者およびメンテナーがどのようにイシューやコントリビューションを管理するかを学びます。
+weight: 999
+default_lang_commit: 722beb5b9ecf8a603b2f5ae6128e3c3509f52fcf
+cSpell:ignore: chalin docsy
+---
+
+このページでは、承認者およびメンテナーが使用するガイドラインと一般的なプラクティスについて説明します。
+
+## オンボーディング {#onboarding}
+
+コントリビュータがドキュメントに対する責任の大きい役割（承認者、メンテナー）を引き受ける場合、既存の承認者およびメンテナーによってオンボーディングが行われます。
```

Comment:
```suggestion
コントリビューターがドキュメントに対する責任の大きい役割（承認者、メンテナー）を引き受ける場合、既存の承認者およびメンテナーによってオンボーディングが行われます。
```

Reply:
@ymotongpoo

Thank you for your review.
I fixed it in a below commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6538/commits/9d3a844ba4c2b78a13608b6b893fa64f141e9822